loader/dd_library_loader.c

```diff
@@ -0,0 +1,355 @@
+/* dd_library_loader extension for PHP */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <Zend/zend_extensions.h>
+#include <php.h>
+#include <stdbool.h>
+
+#include "compat_php.h"
+#include "php_dd_library_loader.h"
+
+static int (*origin_ddtrace_module_startup_func)(INIT_FUNC_ARGS);
+static const zend_module_dep ddtrace_injected_module_deps[] = {ZEND_MOD_OPTIONAL("json") ZEND_MOD_OPTIONAL("standard") ZEND_MOD_OPTIONAL("ddtrace") ZEND_MOD_END};
+static const zend_module_dep *orig_ddtrace_module_deps = NULL;
+static const zend_function_entry *orig_functions;
+
+static bool debug_logs = false;
+static bool force_load = false;
+
+static int php_api_no = 0;
+
+static void ddloader_error_handler(int error_num, zend_string *error_filename, const uint32_t error_lineno, zend_string *message) {}
+
+#define INFO "info"
+#define WARN "warn"
+#define ERROR "error"
+#define LOG(level, format, ...) ddloader_logf(level, format, ##__VA_ARGS__);
+
+static inline void ddloader_logf(const char *level, const char *format, ...) {
+    if (!debug_logs) {
+        return;
+    }
+    char buf[256];
+
+    va_list va;
+    va_start(va, format);
+    vsnprintf(buf, sizeof(buf), format, va);
+    php_printf("[dd_library_loader][%s] %s\n", level, buf);
```

Comment:
Can you please call `_php_error_log(0, msg, NULL, NULL);`? Makes it go to the proper place instead of stdout.